packages/pymedphys/src/pymedphys/deliverydata.py

```diff
@@ -1 +1,4 @@
-from pymedphys_core.deliverydata import DeliveryData
+from pymedphys_deliverydata import DeliveryData
+from pymedphys_deliverydata.dicom import (
```

Comment:
I deleted pymedphys_core and instead made pymedphys_mudensity and pymedphys_deliverydata